docs/pages/config/ssl.md

```diff
@@ -152,13 +152,14 @@ are refused to join the cluster.
 
 | LRL Directive | Description | Default Value |
 |---------------|-------------|---------------|
-| `ssl_client_mode mode` | Can be one of `ALLOW`, `REQUIRE`, `VERIFY_CA` and `VERIFY_HOSTNAME` | `ALLOW` |
+| `ssl_client_mode mode` | Can be one of `ALLOW`<sup>[1](#ssloptfootnote)</sup>, `REQUIRE`, `VERIFY_CA` and `VERIFY_HOSTNAME` | `ALLOW` |
 | `ssl_replicant_mode mode` | Can be one of `ALLOW`, `REQUIRE`, `VERIFY_CA` and `VERIFY_HOSTNAME` | `ALLOW` |
 | `ssl_cert_path path` | Directory containing the server certificate files. Comdb2 searches `server.crt`, `server.key` and `root.crt` for the server certificate, key and trusted CAs respectively | The data directory |
 | `ssl_cert file`| Path to the certificate | `<ssl_cert_path>/server.crt` |
 | `ssl_key file` | Path to the key | `<ssl_cert_path>/server.key` |
 | `ssl_ca file` | Path to the trusted CA certificates | `<ssl_cert_path>/root.crt` |
 
+<a name="ssloptfootnote">[1]</a>: If mode is `ALLOW`, the server will try to open the certificate file in the designated location and will fail to start if it can not be found. For the server configuration, an equivalent mode to `ALLOW` is `OPTIONAL` which behaves in exactly the same way except it will not fail to start the server if certificate file is not found at start time.
```

Comment:
This isn't quite accurate. `OPTIONAL` mode can still fail to proceed if certificate locations are explicitly specified and certificate files could not be found there. In other words, specifying any of `ssl_cert_path`, `ssl_cert`, `ssl_key` and `ssl_ca` implies `ALLOW`.